config/blobs.yml

```diff
@@ -94,7 +94,3 @@ postgres/postgresql-11.22.tar.gz:
   size: 26826810
   object_id: d1f8d34c-b438-44e7-7672-5daea8a6da66
   sha: sha256:6445a4e1533c1e8bb616d4a3784bdc4c0226b541f6f0c8d996d9f27d581d49c3
-redis/7.2.5.tar.gz:
```

Comment:
**question** Is there any reason to keep this reference here?

Reply:
No, when the `packages/redis/*` files are removed, this also needs to be deleted.